### ISSUE TITLE
React Native support & video support

### ISSUE THREAD

**Issue by tmjordan** [mohitk05/react-insta-stories#8]
Hi @mohitkarekar @mohitk05 , first thank you for your amazing project.
I need to implement it in mobile phone, using react native. Is there a compatibility? If no we can work on it.
Also add video support will be a great feature.
I think this lib can become very popular, by allowing a lot of people to use it.
Thanks

**Comment by mohitk05** [comment]
Hello @tmjordan! Glad you liked the project. 
Right now, there is no React Native implementation for this. Would be happy if you'd like to contribute for the same. I'll create a repository and link it here shortly :)
Video support is already added! Please update to the latest version! Use `type: video` inside every story object, while passing a video url!

**Comment by mohitk05** [comment]
Hey @tmjordan, this is the React Native repo: https://github.com/mohitk05/react-native-stories

All relevant updates would happen here. I have started RN development.

**Comment by tmjordan** [comment]
Hi @mohitk05 , ok that's great, are you new to RN development? Anyway there is not a lot of differences between React and React Native, So we just have to adapt your code to the corresponding components in RN.
Let's do it

**Comment by mohitk05** [comment]
@tmjordan yes I am aware of that! I am currently making changes to code wherever necessary, particularly the HTML tags. I will push the code in a day, can you take up porting the CSS animation logic to the RN Animated API? You can find that in Container and Progress components.

**Comment by tmjordan** [comment]
@mohitk05 I dont't have too much experience in animations. But i will give a try, to learn more.

**Comment by trinadhkoya9** [comment]
@mohitk05  when can we see the react native version

**Comment by mayconmesquita** [comment]
+1 for react-native

**Comment by Zahid404** [comment]
Ahhaaann +1 for RN.

**Comment by RishavKumar-3796** [comment]
+1 for react native

**Comment by mohitk05** [comment]
Taking this up soon :)

**Comment by MehmetDT** [comment]
+1 for react native

**Comment by codervss** [comment]
Hey @mohitk05.
Can you please tell me that by what date would the React Native support be added for this?

**Comment by MehmetDT** [comment]
@mohitk05 we are waiting bro <3

**Comment by mohitk05** [comment]
This inspires me :) spent the last night in figuring out some logic stuff, will push refactored code soon, native version too.

**Comment by ChristopherJohnson25** [comment]
@mohitk05 stoked for RN, any updates?

**Comment by ChristopherJohnson25** [comment]
Hi @mohitk05, any updates on the React Native extension?

**Comment by Hirbod** [comment]
+1 for RN

**Comment by ChristopherJohnson25** [comment]
@mohitk05 Do you have a timeline for react native?

**Comment by partitect** [comment]
+ for React Native

**Comment by santoshgistto** [comment]
How about using `React-Native-Web` ?

**Comment by ChristopherJohnson25** [comment]
Any updates on React Native??

**Comment by MehmetDT** [comment]
any updates?*??????

**Comment by tomdowning82** [comment]
Any updates on RN?
